src/hooks/useLocalizedCurrency/useLocalizeCurrency.js

```diff
@@ -1,12 +1,14 @@
 import { useIntl } from 'react-intl';
 import { useStripes } from '@folio/stripes/core';
-import { localizeCurrencyAmount } from '../../components/util/localizeCurrencyAmount';
+import localizeCurrencyAmount from '../../components/util/localizeCurrencyAmount';
```

Comment:
1. rename `src\hooks\useLocalizedCurrency\useLocalizeCurrency.js` to `src\hooks\useLocalizedCurrency\useLocalizedCurrency.js`
2. add index.js file in `src\hooks\useLocalizedCurrency` folder and export it from there
3. export the same at `src\hooks\index.js`
4. adjust the associated imports